utils/src/main/scala/za/co/absa/enceladus/utils/schema/SchemaUtils.scala

```diff
@@ -22,6 +22,45 @@ import scala.util.{Random, Try}
 
 object SchemaUtils {
 
+  /**
+   * Get a field from a text path and a given schema
+   * @param path   The dot-separated path to the field
+   * @param schema The schema which should contain the specified path
+   * @return       Some(the requested field) or None if the field does not exist
+   */
+  def getStructField(path: String, schema: StructType): Option[StructField] = {
```

Comment:
Maybe just `getField`? The 'Struct' part is missing from the other similar methods.